code/build.R

```diff
@@ -633,7 +633,7 @@ if (GENERATE_PMTILES) {
     ), collapse = "; ")
     command_all = paste0(command_start, '"', command_all, '"')
   }
-  responce = system(command_all, intern = TRUE)`
+  responce = system(command_all, intern = TRUE)
```

Comment:
This is a legacy typo from Malcolm, should be `response` not `respoce` [sic] throughout.

```suggestion
  response = system(command_all, intern = TRUE)
```

Reply:
More broadly, we should replace the current code with something like

```r
tippecanoe::make_tiles(x, x.pmtiles, ...)
```

The {tippecanoe} package has yet to be created ; )